contracts/truefi2/BorrowingMutex.sol

```diff
@@ -41,23 +48,19 @@ contract BorrowingMutex is IBorrowingMutex, UpgradeableClaimable {
         emit LockerAllowed(_locker, isAllowed);
     }
 
-    function ban(address borrower) external override onlyAllowedToLock {
-        _lock(borrower, BANNED);
+    function ban(address borrower) external override onlyLocker(borrower) {
```

Comment:
Tracking the different access permissions for `ban()`/`lock()`/`unlock()` is starting to get rather messy:
- only a borrower's locker can ban a borrower, but if a borrower is unlocked then they can't be banned
- only FTLA and LOCA can lock a borrower... but by our design FTLA can't unlock borrowers.
- only FTLA _loans_ or LOCA can unlock a borrower.
- banning and unlocking are nearly identical operations (setting the locker address to a sentinel value). But they have different access permissions.

We should move forward with this change because it makes sense in our current lending exclusion model, but if we find an opportunity to simplify then we should take it. (Follow-up task filed.)